app/presenters/offender_presenter.rb

```diff
@@ -16,28 +16,16 @@ class OffenderPresenter
            :licence_expiry_date, :post_recall_release_date,
            :over_18?, :recalled?, :sentenced?, :immigration_case?, :mappa_level, to: :@offender
 
-  def initialize(offender, responsibility)
+  def initialize(offender)
     @offender = offender
-    @responsibility = responsibility
   end
 
   def pom_responsibility
-    # If this presenter was provided with a responsibility object from
-    # a responsibility override, we will return that, falling back on
-    # asking the offender class to calculate it.
-    if @responsibility.present?
-      if @responsibility.value == Responsibility::PRISON
-        ResponsibilityService::RESPONSIBLE
-      else
-        ResponsibilityService::SUPPORTING
-      end
-    else
-      @offender.pom_responsibility
-    end
+    @offender.pom_responsibility
   end
 
   def responsibility_override?
-    @responsibility.present?
+    @offender.responsibility.present?
```

Comment:
would probably be nicer just to forward this method to @offender

Reply:
@starswan I don't understand what you mean? Are you suggesting that I delegate the responsibility_override? method to the offender object?

Reply:
yes